doc/languages-frameworks/beam.section.md

```diff
@@ -30,6 +30,8 @@ Erlang.mk works exactly as expected. There is a bootstrap process that needs to
 
 For Elixir applications use `mixRelease` to make a release. See examples for more details.
 
+There is also a `buildMix` helper, whose behavior is closer to that of `buildErlangMk` and `buildRebar3`. The primary difference is that `mixRelease` fetches dependencies using `mix deps.get` and builds a release, `buildRebar3` accepts a list of dependencies in the `beamDeps` and compiles the package without buliding a release.
```

Comment:
mixRelease accepts depencies as an argument.
Here I would put.
`The primary difference is that mixRelease makes a release, while buildMix would be used for just building a dependency`.
Open to variations of this if you have strong feelings.
Also I think you meant `buildMix` instead of buildRebar3